bin/proving-service/README.md

```diff
@@ -22,9 +24,12 @@ sudo sha256sum prover.deb | awk '{print $1}' > prover.sha256
 sudo diff prover.sha256 prover.checksum
 sudo dpkg -i prover.deb
 sudo rm prover.deb
+```
 
-sudo chown -R miden-prover /opt/miden-prover
+Edit the configuration file at `/lib/systemd/system/miden-prover.service.d/debian.conf`.
```

Comment:
Should we mention the format of the configuration file? Could be just a reference to one of the sections further in this file.

Reply:
Mentioned in the other comment - the config file is not empty and its based on systemd standard.